assets/css/abstracts/_variables.scss

```diff
@@ -39,3 +39,6 @@ $orange-focus: #fadc90;
 $yellow: #ff0;
 $white: #fff;
 $grey: #bdbdbb;
+
+// Images
+$img-size-medium: rem(262); // need to decide and set other size variables
```

Comment:
I'm not sure what this comment means. Is it still needed?

Reply:
Yes, we need to decide on standard sizes for all types of images.

``` css
// Images
$img-size-small: ?;
$img-size-medium: rem(262);
$img-size-large: ?;
```

I will bring this up in the check-in today.